test/interpreter/function-varpa.spec.ts

```diff
@@ -0,0 +1,26 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {adr, detailedError} from '../testUtils'
+import {ErrorMessage} from '../../src/error-message'
+
+describe('Function VARPA', () => {
+  it('should take at least one argument', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VARPA()'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should calculate variance (population)', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VARPA(2, 3)'],
+      ['=VARPA(2, 3, 4, TRUE(), FALSE(), "1",)'],
+      ['=VARPA(B3:I3)', 2, 3, 4, true, false, 'a', '\'1', null],
+      ['=VARPA(1)'],
```

Comment:
Please split into clear descriptions that we're testing. From the previous comment it can be split into:

```
it('should ignore empty values or text that cannot be coerced to number in references', () => {
it('should evaluate TRUE as 1 and FALSE as 0 (zero)', () => {
```

Any others?